src/gui/tray/UserModel.cpp

```diff
@@ -731,10 +732,11 @@ Q_INVOKABLE void UserModel::openCurrentAccountServer()
         return;
 
     QString url = _users[_currentUserId]->server(false);
-    if (!(url.contains("http://") || url.contains("https://"))) {
+    if (!url.startsWith("http")) {
```

Comment:
I'd prefer if we can match a full scheme here. (in case someone has local handlers that start with `http` but may do other things.Unlikely, but you never know.)

Reply:
Didn't @er-vin say something about it before?

Reply:
@LukasReschke Addressed.